terraform/groups/ecs-service/vault.tf

```diff
@@ -0,0 +1 @@
+./vault-providers/userpass
```

Comment:
This still doesn't appear to be a symbolic link?

Reply:
sorry could you possibly elaborate? I went off the examples in the above links you gave, but I don't know what exactly makes it a symbolic link so didn't realise it was not one. Thanks!

Reply:
I think the idea is that the file `vault.tf` can be/should be a symbolic link to the file `./vault-providers/userpass`.